src/users/Admin.java

```diff
@@ -0,0 +1,22 @@
+package users;
+
+public class Admin extends AbstractUser {
+    public Admin(String name) {
+        super(name);
+    }
+
+    @Override
+    public void printRole() {
+        System.out.println("My role is ADMIN");
+    }
+
+    @Override
+    public void getTicket() {
+
+    }
```

Comment:
is not the best practice. In this case, you can create only a few interfaces and implement only the necessary ones

ang in methet getTicket I expect not void.

Reply:
done